public/javascripts/studentControllers.js

```diff
@@ -28,17 +28,26 @@ var studentController = function($scope,  $http, $state) {
 };
 
 var addEntryController = function($scope, $http, $location) {
-  // $scope.minSlider = {
-  //   value: 10
-  // };
+  $scope.newEntry = {engageContent: 5, engagePeer: 5};
   $scope.submitNewEntry = function(student) {
     $http.post('api/student/newEntry/' + student._id, $scope.newEntry)
     .then(function successCallback(response) {
       $scope.entrySubmittedMsg = response.data.msg;
+      $scope.newEntry = {engageContent: 5, engagePeer: 5};
```

Comment:
This line basically nulls the form and resets the sliders to this value. Adding just the null messes up with the sliders and adding null along with this line isn't doing anything new